src/gateway/kong-enterprise/secrets-management/advanced-usage.md

```diff
@@ -44,7 +44,7 @@ This lets you drop the configuration from environment variables and query argume
 {vault://my-env-vault/my-secret-config-value}
 ```
 
-For more information, see the section on the [Vaults entity](#vaults-entity).
+For more information, see the section on the [Vaults entity](#vaults-object).
```

Comment:
@raoxiaoyan For this link, are you wanting to link to the Admin API page? I think this link originally linked to the "Vaults Entity" section on this same page.

Reply:
@cloudjumpercat I need to keep the link of this vaults to the same specification as the links of other entities. Because the docs of the new vaults in KM 3.1 cannot link to the corresponding page correctly